iaso/tests/api/test_profile_logs.py

```diff
@@ -560,3 +696,68 @@ def test_filters(self):
 
         results = data["results"]
         self.assertEquals(results, [])
+
+    def test_retrieve(self):
```

Comment:
I would add more tests for retrieve:

- retrieve on created instance without update (to spot the crash I mentioned before)
- retrieve without authentication
- retrieve without permissions
- retrieve for unknown log ID